app/src/main/res/layout/fragment_client_settings.xml

```diff
@@ -7,9 +7,6 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:background="@color/primary_dark"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/bf6320090a88ce6cc95c72470de2d4697673ea27/app/src/main/res/layout/fragment_client_settings.xml#L9"></span>Possible overdraw: Root element paints background <code>@color/primary_dark</code> with a theme that also paints a background (inferred theme is <code>@style/AppTheme</code>)</td>
    </tr>
  </tbody>
</table>